src/core/db/repository/base.py

```diff
@@ -30,12 +30,11 @@ async def get(self, _id: int) -> DatabaseModel:
             raise NotFoundException(object_name=self._model.__name__, object_id=_id)
         return db_obj
 
-    @auto_commit
     async def create(self, instance: DatabaseModel) -> DatabaseModel:
         """Создает новый объект модели и сохраняет в базе."""
         self._session.add(instance)
         try:
-            self._session.commit()
+            auto_commit()

```

Comment:
Очень не приятная функция. Она в любом случае должна делать коммит, иначе в ней нет смысла. Оставь как было. Но добавь аргумент `commit` в саму функцию - не через декоратор
```suggestion
            self._session.commit()
```